site-builder/src/main.rs

```diff
@@ -270,6 +284,29 @@ pub(crate) struct Config {
 }
 
 impl Config {
+    pub fn load_multi_config(path: impl AsRef<Path>, context: Option<&str>) -> Result<Self> {
+        let mut multi_config =
+            serde_yaml::from_str::<MultiConfig>(&std::fs::read_to_string(path)?)?;
+
+        let context = context.unwrap_or_else(|| &multi_config.default_context);
+        tracing::info!(?context, "using context");
```

Comment:
```suggestion
        tracing::info!(?context);
```

Otherwise it will show up like this:
```
2025-03-26T11:09:04.340394Z  INFO site_builder: using context context="testnet"
```